lib/ES/BaseRepo.pm

```diff
@@ -22,7 +22,8 @@ sub new {
         $reference_subdir =~ s|/$||;
         $reference_subdir =~ s|:*/*\.git$||;
         $reference_subdir =~ s/.*[\/:]//g;
-        $reference_dir = $args{reference}->subdir("$reference_subdir.git");
+        #$reference_dir = $args{reference}->subdir("$reference_subdir.git");
+        $reference_dir = $args{reference}->subdir("elastic-$reference_subdir");
```

Comment:
If we merge this in as is, it will break existing builds (or at least really slow them down since it won't use any of the existing cached repos.

Reply:
oh yes - good catch! I forgot about that one!!!